pkg/ui/ui.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/server/serverpb"
 	"github.com/cockroachdb/cockroach/pkg/settings"
 	"github.com/cockroachdb/cockroach/pkg/settings/cluster"
+	_ "github.com/cockroachdb/cockroach/pkg/ui/settings" // Import the settings package to register UI-related settings for doc generation.
```

Comment:
unrelated?

Reply:
Actually, the ui settings were previously only imported in the upgrades that are deleted in this PR, so it resulted in them being removed from the auto generated settings docs. Adding this in here is done to make sure the settings stay documented.